### PR TITLE
sci-mathematics/octave-6.4.0: add missing dependencies

### PR DESCRIPTION
**Pull request by CHN-beta**
Bug: https://bugs.gentoo.org/837752
Signed-off-by: Haonan Chen <chn@chn.moe>

### CODE REVIEW

File: sci-mathematics/octave/octave-6.4.0-r1.ebuild [hunk 1]
```diff
@@ -38,6 +39,7 @@ RDEPEND="
 	glpk? ( sci-mathematics/glpk:0= )
 	gnuplot? ( sci-visualization/gnuplot )
 	gui? (
+		dev-qt/qtchooser
```
Comment (a17r) [inline sci-mathematics/octave/octave-6.4.0-r1.ebuild:42]: Do NOT depend on qtchooser. Fix the build system or use qmake-utils.

File: sci-mathematics/octave/octave-6.4.0-r1.ebuild [hunk 2]
```diff
@@ -29,6 +29,7 @@ RDEPEND="
 	app-text/ghostscript-gpl
 	sys-apps/texinfo
 	dev-libs/libpcre:3=
+	media-gfx/graphicsmagick
```
Comment (a17r) [inline sci-mathematics/octave/octave-6.4.0-r1.ebuild:32]: This is unlikely a hard-dependency on graphicsmagick. See also https://bugs.gentoo.org/314431

**Review by a17r — CHANGES_REQUESTED** [review]

File: sci-mathematics/octave/octave-6.4.0-r1.ebuild (hunk 2 above)
Comment (CHN-beta) [reply sci-mathematics/octave/octave-6.4.0-r1.ebuild]: Thank you for your suggestion. I have force pushed a new ebuild.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1: 

I changed the ordering of the inherits, usually we put them alphabetically. I also fixed the `UnnecessarySlashStrip` warning, which contrary to what the qa-bot claims is not actually introduced in this Pull Request but was already present in the repository.

**Pull request closed without merging**